File: Mobile/lib/Modules/operation.dart
class OperationMoudle {
  int? id;
  int? scheduleid;
  int? adminid;
  int? usertypeid;
  int? status;
  String? notes;
  String? answers;
  PatientInfo? patientInfo;

  OperationMoudle(
      {this.id,
      this.scheduleid,
      this.adminid,
      this.usertypeid,
      this.status,
      this.notes,
      this.answers,
      this.patientInfo});

  OperationMoudle.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    scheduleid = json['scheduleid'];
    adminid = json['adminid'];
    usertypeid = json['usertypeid'];
    status = json['status'];
    notes = json['notes'];
    answers = json['answers'];
    patientInfo = json['patientInfo'] != null
        ? PatientInfo.fromJson(json['patientInfo'])
        : null;
  }
}

class PatientInfo {
  int? id;
  String? name;
  int? age;

  PatientInfo({this.id, this.name, this.age});

  PatientInfo.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    age = json['age'];
  }
}

File: Api/SecurityDoctor/SecurityDoctor/DataBase/Operation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;
using System.Xml.Serialization;

namespace SecurityDoctor.DataBase
{
    [Table("T_Operation")]
    public class Operation
    {
         public int id { get; set; }
         public int scheduleid{get;set;}
        public int adminid { get; set; }
        public int usertypeid { get; set; }

        public int status { get; set; }
        public string notes { get; set; }
        public string answers { get; set; }
        [JsonIgnore]
        [XmlIgnore]
        public List<OperationImage> OperationImage { get; private   set; }
        [JsonIgnore]
        [XmlIgnore]
        public Schedule Schedule { get; private set; }
    }
}
